docs/check.go

```diff
@@ -145,11 +145,16 @@ func parseCategoryNames(configPath string) (StringSet, error) {
 			continue
 		}
 		if found {
-			if !strings.HasPrefix(l, categoryPrefix) {
-				found = false
+			if strings.HasPrefix(l, commentPrefix) {
+				// skip commented lines
 				continue
 			}
-			categoryNames.Add(strings.TrimPrefix(l, categoryPrefix))
+			if strings.HasPrefix(l, categoryPrefix) {
+				categoryNames.Add(strings.TrimPrefix(l, categoryPrefix))
+				continue
+			}
+			// stop parsing if line is neither comment or category
+			found = false
```

Comment:
would it just be easier to add a yaml parsing command line tool in the CI container, and then use that to query for this?
ex:  https://kislyuk.github.io/yq/

Reply:
pip -> python -> i have nothing more to say
we don't have python in the image, i would have given more consideration if it was already present and used for something else

Reply:
but the next search result:
https://github.com/mikefarah/yq

Reply:
I am ok adding to the CI image. It will help make the code here more readable

Reply:
ok adding `python + pip + yq` or `wget yq && mv /usr/local/bin/`?

Reply:
Isn't go installed? Can we not use `GO111MODULE=on go get github.com/mikefarah/yq/v3` from the link you gave above?

Reply:
Either way, I'm good with adding the go-based one